flink-core/src/main/java/org/apache/flink/types/Pair.java

```diff
@@ -33,6 +34,7 @@
  * @param <U> Type of the pair's first element.
  * @param <V> Type of the pair's second element.
  */
+@PublicInterface
 public abstract class Pair<U extends Key<U>, V extends Key<V>> implements Key<Pair<U, V>> {
```

Comment:
is this the class mentioned in FLINK-2344? https://issues.apache.org/jira/browse/FLINK-2344